components/AvalancheCenterSelector.tsx

```diff
@@ -68,24 +68,29 @@ const AvalancheCenterCard: React.FunctionComponent<AvalancheCenterCardProps> = (
   }
 
   return (
-    <TouchableOpacity style={styles.item} onPress={() => navigation.navigate('avalancheCenterHome', {center_id: center_id, date: date})}>
+    <TouchableOpacity
+      style={styles.item}
+      onPress={() => {
+        // We need to clear navigation state to force all screens from the
+        // previous avalanche center selection to unmount
+        navigation.reset({
+          index: 0,
+          routes: [{name: 'Home', params: {center_id}}],
+        });
+      }}>
       <AvalancheCenterLogo style={styles.logo} center_id={center_id} />
       <Text style={{textAlignVertical: 'center'}}>{avalancheCenter.name}</Text>
     </TouchableOpacity>
   );
 };
 
-export interface AvalancheCenterSelectorProps {
-  date: string;
-}
-
-export const AvalancheCenterSelector: React.FunctionComponent<AvalancheCenterSelectorProps> = ({date}: AvalancheCenterSelectorProps) => {
+export const AvalancheCenterSelector = () => {
   return (
     <SectionList
       style={styles.container}
       sections={center_idsByType}
       keyExtractor={item => item}
-      renderItem={({item}) => <AvalancheCenterCard center_id={item} date={date} />}
```

Comment:
Why'd we lose the date?

Reply:
After selecting a center, we now clear the nav stack and go back to the very first view (`Home`) which references the top-level `defaultDate`